lighthouse-core/test/network-records-to-devtools-log.js

```diff
@@ -63,7 +63,8 @@ function getRequestWillBeSentEvent(networkRecord, index) {
         initialPriority: networkRecord.priority || 'Low',
         isLinkPreload: networkRecord.isLinkPreload,
       },
-      timestamp: networkRecord.redirectResponseTimestamp || networkRecord.startTime || 0,
+      timestamp: networkRecord.redirectResponseTimestamp || networkRecord.rendererStartTime ||
```

Comment:
no test in this PR explicitly sets `rendererStartTime`, but by default `network-records-to-devtools-log` checks that round-tripping networkRecords->DevtoolsLog->networkRecords reproduces the input and some tests (like `uses-http2`) use slightly altered but real network records that have an actual `rendererStartTime` that now needs to make its way to the output